charts/consul/test/unit/gossip-encryption-autogenerate-job.bats

```diff
@@ -38,26 +38,3 @@ load _helpers
   [ "$status" -eq 1 ]
   [[ "$output" =~ "If global.gossipEncryption.autoGenerate is true, global.gossipEncryption.secretName and global.gossipEncryption.secretKey must not be set." ]]
 }
-
-@test "gossipEncryptionAutogenerate/Job: fails if global.gossipEncryption.autoGenerate=true and global.gossipEncryption.secretName is set" {
```

Comment:
This test is now invalid due to requiring both secretKey + Name to be specified in server-statefulset

Reply:
why are they invalid? I don't think it's the same behavior. The server statefulset will fail if you've provided only one of secretName/secretKey. These tests are checking that you haven't set them when autoGenerate is true.